nixos/modules/hardware/video/nvidia.nix

```diff
@@ -180,6 +189,25 @@ in
     environment.systemPackages = [ nvidia_x11.bin nvidia_x11.settings ]
       ++ lib.filter (p: p != null) [ nvidia_x11.persistenced ];
 
+    systemd.services = let
+      nvidiaService = type: {
```

Comment:
I don't really think this should be called type, maybe `sleepState`, like in `systemd-suspend.service`?